src/models/reposyncstate.ts

```diff
@@ -139,13 +139,28 @@ export class RepoSyncState extends Model {
 			}
 		});
 
```

Comment:
This is a one time process to remove duplicate records from the table,  can't we add it as a DB migration script?

Reply:
I think until we have the fix in it will be an ongoing issue, so migration script would just clear the current duplicates but then we will continue to get more in between times??

Reply:
We can, but this is also the code that added the duplicates in the first place because of a partial backfill.  I figured removing it at the source first, then doing a DB script (that'll be another PR) to remove the duplicates in the DB is the best way to go.  Then I need to update the DB schema so that this can never happen ever again :)